### PR TITLE
Update comments

### LINKED ISSUE #659: Use new Dhall syntax for overrides
> I just realized we have a nicer syntax for that snippet in the newer Dhall/Spago versions, now you should be able to write that as:

```dhall
upstream
  with argonaut.version = "v7.0.0"
  with other-lib.version = "v1.2.3"
```

### PR DESCRIPTION
**Pull request by JordanMartinez**
### Description of the change

Fixes #659

### Checklist:

- [x] Added the change to the "Unreleased" section of the changelog
- [x] Added some example of the new feature to the `README`
- [ ] Added a test for the contribution (if applicable)

**P.S.**: the above checks are not compulsory to get a change merged, so you may skip them. However, taking care of them will result in less work for the maintainers and will be much appreciated 😊

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -477,13 +477,10 @@ In this case, we can just change the override to point to some commit of our for
...
 
 ```haskell
-let overrides =
-    { simple-json =
-          upstream.simple-json
-       // { repo = "https://github.com/my-user/purescript-simple-json.git"
-          , version = "701f3e44aafb1a6459281714858fadf2c4c2a977"
-          }
-    }
+let upstream = --
```
Comment (f-f) [inline README.md:480]: I kind of like leaving out the URL and hoping that people don't try to copypaste this, but I'm wondering if we can find a way to make this self contained? :thinking:

Comment (JordanMartinez) [reply README.md:480]: I could clarify that via a more detailed comment
```dhall
let upstream = -- put package set release URL here
```
When people do `spago init`, the file itself will indicate how that works.

Comment (JordanMartinez) [reply README.md]: How does latest commit's changes look?

Comment (f-f) [reply README.md]: Looks great, cheers :clap:

**Comment by f-f** [comment]
This is good to go, but it looks like there's a test failing - I suspect we have fixtures trying to match on the content of the template

**Comment by JordanMartinez** [comment]
It's caused by this line:
```diff
-      dhallSource `shouldSatisfy` (Text.isInfixOf "let overrides =") -- some dhall stuff is present
+      dhallSource `shouldSatisfy` (Text.isInfixOf "in upstream") -- some dhall stuff is present
```

**Comment by f-f** [comment]
Thank you! :100:

File: README.md [hunk 2]
```diff
@@ -477,13 +477,10 @@ In this case, we can just change the override to point to some commit of our for
...
-    { simple-json =
-          upstream.simple-json
-       // { repo = "https://github.com/my-user/purescript-simple-json.git"
-          , version = "701f3e44aafb1a6459281714858fadf2c4c2a977"
-          }
-    }
+let upstream = -- <package set URL here>
+in  upstream
+  with simple-json.repo = "https://github.com/my-user/purescript-simple-json.git"
+  with simple-json.verison = "701f3e44aafb1a6459281714858fadf2c4c2a977"
```
Comment (thomashoneyman) [inline README.md:483]: Typo here -- `version`

Comment (JordanMartinez) [reply README.md:483]: Dah! I'm pretty sure my fingers have muscle memory now in spelling this word wrong!

Comment (JordanMartinez) [reply README.md:483]: Fixing in #662

**Pull request merged**